First/src/MediumDifficultyAssignmnets/AnnualSalary.java

```diff
@@ -0,0 +1,45 @@
+package firstPackage.First.src.MediumDifficultyAssignmnets;
+
+public class AnnualSalary {
+    private static float tax;
+    private static float taxableAmount;
+
+    public float displayIncomeTax(float income) {
+        int Incomeslab = (income < 500000) ? 1 : (500000 <= income && income <= 700000)
+                ? 2 : (700000 < income && income <= 1000000) ? 3 : (income <= 1500000) ? 4 : 5;
```

Comment:
Please verify the conditions and change accordingly

Reply:
Done !